plugins/follow_me/follow_me.h

```diff
@@ -0,0 +1,170 @@
+#pragma once
+
+#include <iostream>
+#include <cmath>
+#include <functional>
+
+namespace dronecore {
+
+class FollowMeImpl;
+
+/**
+ * @brief FollowMe plugin is used to make your drone to follow your Ground Control Station (Phone/Tab). <br>
```

Comment:
"FollowMe plugin is used to make your drone to follow your Ground Control Station (Phone/Tab). <br>" TO
"This class is used to manage the vehicle FollowMe mode."

Reply:
Thanks @hamishwillee